src/components/TheDrawer.vue

```diff
@@ -1,12 +1,16 @@
 <script setup lang="ts">
 import { Drawer, DrawerContent } from "@progress/kendo-vue-layout";
 import { useLocalStorage } from "@vueuse/core";
-import { computed } from "vue";
+import { computed, ref } from "vue";
+import { useRouter } from 'vue-router';
+
+const router = useRouter();
 
 const expanded = useLocalStorage("vue-forge-drawer-expanded", true);
 const expandedIcon = computed(() =>
   expanded.value ? "k-i-arrow-chevron-left" : "k-i-arrow-chevron-right"
 );
+const selectedIndex = ref(0);
```

Comment:
Shouldn't `selectedIndex` be computed from current route path? I think selectedIndex would still show up as Boards if I click on Templates link and refresh the page. Or am I wrong?